Amazon.Ion.ObjectMapper/IonObjectSerializer.cs

```diff
@@ -124,6 +120,118 @@ public override void Serialize(IIonWriter writer, object item)
             writer.StepOut();
         }
 
+        private bool DeserializeProperty(PropertyInfo property, IIonReader reader, IonType ionType, ref object deserialized)
+        {
+            if (IsReadOnlyProperty(property))
+            {
+                // property.SetValue() does not work with a readonly property.
+                // logic for handling deserializing readonly properties happens during field processing
+                // when we detect backing fields for the property.
+                return false;
+            }
+
+            deserialized = ionSerializer.Deserialize(reader, property.PropertyType, ionType);
+            if (options.IgnoreDefaults && deserialized == default)
```

Comment:
You can just return the inverted value here instead of `if (return false) else return true`

Reply:
Done.